packages/react/src/app-layout/AppLayoutHeaderBrand.tsx

```diff
@@ -104,3 +221,21 @@ function AppLayoutHeaderBrandBadge({
 		</Box>
 	);
 }
+
+const DividingLine = ({
+	dividerPosition,
+}: Pick<AppLayoutHeaderBrandProps, 'dividerPosition'>) => (
+	<Box
```

Comment:
Can you use the border props like the normal Header one?

Reply:
This is just copy/paste from the existing one in this file. Are you suggesting to change both?

Reply:
It's not quite a copy-paste. Here's Header's:

```
const DividingLine = ({
	dividerPosition,
}: Pick<HeaderBrandProps, 'dividerPosition'>) => (
	<Box
		borderRight
		css={packs.print.hidden}
		display={{
			xs: 'none',
			sm: dividerPosition === 'between' ? 'block' : undefined,
			lg: 'block',
		}}
	/>
);
```

Reply:
- [ ] Use this component for the existing dividing line when there is no second logo.
- [ ] Use props over `css` prop values

Reply:
Wasn't able to eliminate the `css` prop as the custom styles needed overriding more than was worth it in the end.

Reply:
You can still eliminate the border ones though?

Reply:
The props aren't setup this way, unfortunately. We have `borderLeft` and `borderRight` but not the other ones. And these are booleans that render a fixed style. That works for the `Header` but in `AppLayoutHeader` we have this variation with greater width and a different colour.

Reply:
Just using the `borderRight` prop instead of these border CSS properties should do the trick, just like in Header. It would appear that the existing is a bug, because (most?) browsers' default border width is 1.5px, which is not one of our allowed values, and I think purely done by accident

Reply:
lol, Safari's default border width is 3px. There's no way we want that 
<img width="734" alt="image" src="https://github.com/agriculturegovau/agds-next/assets/853552/64e9a657-5a43-41cf-9bee-3d55f131f59d">

Reply:
Interesting! The designs are suggesting 1px and the height looks like a bug, too.

- [ ] use `borderRight` (or `borderLeft`) for divider
- [ ] set divider height to the `LOGO_HEIGHT`

Reply:
Design is 1px wide 
<img width="677" alt="image" src="https://github.com/agriculturegovau/agds-next/assets/853552/79fbd5a6-e8b3-4d81-baf9-fbd1df5c9326">